src/components/Explore/utils.ts

```diff
@@ -430,6 +435,21 @@ export const exploreMetricData: {
       },
     ],
   },
+  [ExploreMetric.WEEKLY_DEATHS_PER_100K]: {
+    title: 'Weekly reported deaths per 100k',
+    name: 'Weekly reported deaths per 100k',
+    chartId: 'weekly_reported_deaths_per_100k',
+    dataMeasure: DataMeasure.INTEGER,
+    yAxisDecimalPlaces: 0,
+    seriesList: [
+      {
+        label: 'Weekly reported deaths per 100k',
```

Comment:
This label is showing up as the legend underneath the weekly death per 100k chart in the additional metrics chart group. This is because if it is an explore metric (see `Legend` component in `AddedMetricChartFooter.tsx`), where legends are shown by default. I think the legend in this case is a little bit redundant, but perhaps this adjustment can be made in a follow-up PR.

Reply:
Yeah, I'm not too worried about it.  Per the comment above I would remove "reported" here and below though.  And figure out the "per 100k" thing.